src/main/webapp/profile.html

```diff
@@ -82,13 +82,12 @@ <h1 class="text-center">Your Profile</h1>
         <div class="form-row">
           <div class="col-md-12 mb-3">
             <label for="linkedin-field">LinkedIn URL</label>
-            <input type="text" readonly class="form-control-plaintext editable" id="linkedin-field" placeholder="https://www.linkedin.com/" value="">
+            <input type="text" readonly class="form-control-plaintext editable" id="linkedin-field" placeholder="https://www.linkedin.com/" value="" name="linkedin">
           </div>
         </div>
         <div class="form-group row">
           <div class="col-sm-10">
-            <!-- TODO: Have this update button actually update profile info to Datastore. -->            
-            <button type="submit" id="update-button" hidden class="btn btn-primary">Update</button>
+            <button class="btn btn-primary" id="update-button" type="submit" hidden>Update Profile</button>
```

Comment:
There's no function being called here which makes me suspicious of the PR title.

Reply:
lt's a form that submits to a servlet. l tested, dw.